bodhi/tests/server/test_models.py

```diff
@@ -742,6 +742,15 @@ class TestBuild(ModelTest):
     klass = model.Build
     attrs = dict(nvr=u"TurboGears-1.0.8-3.fc11")
 
+    def do_get_dependencies(self):
```

Comment:
I recommend calling this ``_do_real_get_fetch_dependencies`` 😄 (I see you don't have a choice in the matter, though)

Reply:
Hahah